Cloud Enterprise/Getting Started Examples/aws/terraform/main.tf

```diff
@@ -1,19 +1,19 @@
+terraform {
```

Comment:
f this works with TF .12, I wouldn't require any version to be enforced
TF had some well known breaking changes from .11 to .12, but that not the case usually (maybe want to just commend out 'written with terraform vX.X just for reference)

Reply:
I didn't think the changes were backwards compatible, so wanted to ensure at least `>=0.12` ...probably don't need the `.19` addition. 

Does something smart happen if you trying to run a 0.12 config through v0.11?

Reply:
it probably tells you to upgrade haha
>=0.12 is good, just didn't want to enforce a specific version